### DISCUSSION TITLE
Ecliptic Longitude zero

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#2142] [Q&A]
As defined, when Ecl. long is zero, Sun should be at RA 0h00m00s, but in Stellarium we see
when I type 2022-03-20 on date

   RA(on date) = 0h00m00.00s, then Ecl.long. (on date) =+359º59'58.5"

or when

   RA(on date) = 0h00m00.09s, then Ecl.long. (on date) =+360º00'00.0" and few seconds later:
   RA(on date) = 0h00m00.09s, then Ecl.long. (on date) =+0º00'00.0"

In the 1st place, shouldn't 360º be replaced by 0º? And second, at vernal point RA=Ecl.long=0?

assuming precision goes beyond the seconds mark, maybe it's a problem with rounding, but it does hurt my eyes. Thank you and best regards!

**Comment by gzotti** [comment]
The *geocentric* sun should be at RA=&lambda;=0. Switch off topocentric correction. The displayed precision should meanwhile be mostly applicable, for the present time give or take about 0.1'' for the planets and about 0.5'' for the Moon. See appendix F in the User Guide. 
A display of 360°0'0'' means "just below" (a fraction of an arcsecond short of) 360°. It may look strange but is correct.

**Reply by diegorodriguezperegrin** [reply]
Yes, I have checked  RA=λ=0 with topocentric coordinates off and yes, I can see 0h00m00.00s then +360º00'00.0", that would mean there's a bit under 360º. Blame roundings. Many thanks and have a happy 2022 in whatever coordinates systems you wish!
